### PR TITLE
chore(upgrade): v1.5.0 to v1.6.0

### PR DESCRIPTION
**Pull request by enddynayn**
- Upgrade Polkadot-sdk to v.1.6.0.
- Update weights to reflect the new version.

Notable Changes:
- [Rename ExportGenesisStateCommand to ExportGenesisHeadCommand ](https://github.com/paritytech/polkadot-sdk/pull/2331)

For more details, please refer to:

[Release Notes](https://github.com/paritytech/polkadot-sdk/releases/tag/polkadot-v1.6.0)
[Release Analysis](https://forum.polkadot.network/t/polkadot-release-analysis-v1-6-0/5855/2)

#1841

### CODE REVIEW

File: runtime/common/src/fee.rs [hunk 1]
```diff
@@ -77,7 +77,7 @@ mod test {
 
 		assert_eq!(p, 100_000);
 
-		assert!(q >= 65_000_000);
+		assert!(q >= 50_000_000);
```
Comment (shannonwells) [inline runtime/common/src/fee.rs:80]: yaaas

**Review by shannonwells — APPROVED** [review]
Seems good to go.  I did not look through the substrate pallet weight changes.  I did look through our custom pallet weight changes.  The biggest differences that I noticed were lowered weights.

File: Cargo.toml [hunk 2]
```diff
@@ -53,52 +53,53 @@ serde = { version = "1.0", default-features = false }
...
-pallet-collator-selection = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.5.0", default-features = false }
-parachain-info = { package = "staging-parachain-info", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.5.0", default-features = false }
+cumulus-pallet-aura-ext = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.6.0", default-features = false }
+cumulus-pallet-parachain-system = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.6.0", default-features = false }
+cumulus-pallet-session-benchmarking = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.6.0", default-features = false }
+cumulus-primitives-aura = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.6.0", default-features = false }
+cumulus-primitives-core = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.6.0", default-features = false }
+cumulus-primitives-parachain-inherent = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.6.0" }
```
Comment (wilwade) [inline Cargo.toml:98]: With the change to `cumulus-client-parachain-inherent` I believe we can remove `cumulus-primitives-parachain-inherent` entirely.

**Review by wilwade — APPROVED** [review]
One non-blocking unused dependency removal and the e2e test needing a decreased amount due to the lower cost

- Reviewed code

File: Cargo.toml (hunk 2 above)
Comment (enddynayn) [reply Cargo.toml:98]: Nice catch. That seems to be correct.

**Review by mattheworris — APPROVED** [review]
- Read through changes
Nice work!
🚢 it!

**Pull request merged**